### DISCUSSION TITLE
Can flopy be used to simulate temperature transport, conductivity, spontaneous potential, etc.?

### DISCUSSION THREAD

**Discussion by Alvin-prc** [modflowpy/flopy#1792] [Q&A]
I only know how to use flopy to simulate water flow and solute transport. I would like to know whether it can be used to simulate multiple physical fields, such as temperature, conductivity, spontaneous potential, etc. If so, which modules in flopy correspond to?

**Comment by mbakker7** [comment]
Maybe these help:
AUSGS report with a nice description about using SEAWAT for heat transport: https://pubs.er.usgs.gov/publication/tm6A22
A flopy example with heat transport can be found here: https://modflow6-examples.readthedocs.io/en/latest/_notebooks/ex-gwt-hecht-mendez.html

**Reply by Alvin-prc** [reply]
Thank you

**Comment by qinghe-gm** [comment]
Partly related to this question, I also wonder can mf6 simulate groundwater flows from land to ocean? (Currently, the typical way is to set all the ocean grids to have a constant water head)
